src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier implements SideCalculator {
```

Comment:
Remove this interface, it doesn't make much sense. You could have a FigureSupplier interface with definition of its public methods - getrandomfigure and getdefaultfigure and this class would implement it, but it's not necessary